app/controllers/organizations/organization_members_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Organizations
+  class OrganizationMembersController < BaseController
+    PERMITTED_ORGANIZATION_MEMBER_PARAMS = [
+      :first_name, :last_name, :phone, :organization_role,
+      user_attributes: %i[email id]
+    ].freeze
+
+    before_action :set_organization_member, only: %i[edit update destroy]
+
+    helper UsersHelper
+
+    def index
+      @organization_members = helpers.current_organization_admin.organization.organization_members.includes(:user)
```

Comment:
What is the usage of the `.includes(:user)`?

Reply:
Preloading